### PR TITLE
Make sure we do not use SP1 node for cinder-volume

### PR DESCRIPTION
**Pull request by jsuchome**
See e.g. https://ci.suse.de/job/openstack-mkcloud/31710/

### CODE REVIEW

**Comment by jsuchome** [comment]
It would be better to take SP2 node out from that `$unclustered_nodes`, but this is faster and IMHO sufficient. Also, `$sles12plusnode` is the last SP2 node so it likely is also not part of cluster

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2974,6 +2974,10 @@ function custom_configuration()
                 local cinder_volume
                 # fetch one of the compute nodes as cinder_volume
                 cinder_volume=`printf "%s\n" $unclustered_nodes | tail -n 1`
+                # make sure we do not use SP1 node for cinder
+                if [ -n "$deployceph" -a iscloudver 7plus ] ; then
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: no hard opinion, but IIRC we were moving from `[` to `[[`:

`[[ -n "$deployceph" && iscloudver 7plus ]]  && cinder_volume=$sles12plusnode`

Also IIUC ... is this doing the opposite? If deployceph and is a SOC7, cinder_volume is a node with target=suse-12.1 [1]

[1] https://github.com/jsuchome/automation/blob/master/scripts/qa_crowbarsetup.sh#L759

Comment (rhafer) [inline scripts/qa_crowbarsetup.sh]: > If deployceph and is a SOC7, cinder_volume is a node with target=suse-12.1

@aplanas Seems you were looking at an old version of the code ( @jsuchome 's master branch is outdated). The current master picks an SP2 node when running on cloud 7

**Comment by rhafer** [comment]
+1

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: Ops! I see!

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**